pkg/users/account-create-dialog.js

```diff
@@ -314,19 +411,30 @@
                     caption: _("Create"),
                     style: "primary",
                     clicked: () => {
-                        return passwd_check(false, state.real_name, state.user_name, state.password, state.password_confirm, state.locked, state.change_passw_force);
+                        return passwd_check(false, false, state.real_name, state.user_name, state.password, state.password_confirm, state.locked, state.home_dir, state.change_passw_force);
                     },
-                    disabled: state.confirm_weak
+                    disabled: state.confirm_weak || state.home_dir_exists
                 }
             ]
         };
+        if (state.home_dir_exists) {
+            footer.actions.push(
+                {
+                    caption: _("Create and change ownership of home directory"),
+                    style: "warning",
+                    clicked: () => {
+                        return passwd_check(false, true, state.real_name, state.user_name, state.password, state.password_confirm, state.locked, state.home_dir);
+                    }
+                }
+            );
+        }
         if (state.confirm_weak) {
             footer.actions.push(
                 {
                     caption: _("Create account with weak password"),
                     style: "warning",
                     clicked: () => {
-                        return passwd_check(true, state.real_name, state.user_name, state.password, state.password_confirm, state.locked);
+                        return passwd_check(true, false, state.real_name, state.user_name, state.password, state.password_confirm, state.locked, state.home_dir);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230317-103941-df531fdc-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)